renderer.php

```diff
@@ -659,7 +827,7 @@ public function statistics_table_for_ratingallocate(ratingallocate $ratingalloca
 
         $cell = new html_table_cell();
         $usersinchoice = $ratingallocate->get_raters_in_course();
-        $cell->text = count($usersinchoice) - count($memberships);
+        $cell->text = count($ratingallocate->get_undistributed_users());
```

Comment:
`'unassigned' => count($usersinchoice) - count($memberships))));` in line 858 needs to be changes as well.